.gitignore

```diff
@@ -1,3 +1,5 @@
+.DS_Store
+.idea/

```

Comment:
I 100% disagree with this, but if that's something the maintainer wants I'd be willing to oblige in this case.